docs/ref/django-admin.txt

```diff
@@ -963,6 +963,21 @@ beyond those apps. Same as ``--list``, applied migrations are marked by an
 ``[X]``. For a verbosity of 2 and above, all dependencies of a migration will
 also be shown.
 
+sql
+---
+
+.. django-admin:: sql
+
+Prints the SQL statements to create a new database from scratch.
+
+The output of this tool *will not* match the SQL emitted by migrations, and
+*should not* be used for any sort of production or development database
+creation, as it will be incompatible with migrations. It is intended for
+testing and information use only.
```

Comment:
Maybe it could be usefull to explain what might be different between the SQL from migrations and the one from these command ? (as far as I can tell, this would be the SQL generated by the same project if we threw away all the migrations and then called makemigrations, right ?). I'd say it's worth expliciting that RunSQL and RunPython elements are lost in the process.